spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigUtils.java

```diff
@@ -34,15 +38,26 @@
 	private Fabric8ConfigUtils() {
 	}
 
-	public static String getApplicationNamespace(KubernetesClient client, String configNamespace,
+	public static String getApplicationNamespace(KubernetesClient client, String namespace,
 			String configurationTarget) {
-		if (!StringUtils.hasLength(configNamespace)) {
+		if (!StringUtils.hasLength(namespace)) {
 			LOG.debug(configurationTarget + " namespace has not been set, taking it from client (ns="
 					+ client.getNamespace() + ")");
-			configNamespace = client.getNamespace();
+			namespace = client.getNamespace();
 		}
 
-		return configNamespace;
+		return namespace;
+	}
+
+	public static String getApplicationNamespace(KubernetesClient client, String namespace) {
```

Comment:
this was `getNamespace`, renamed it to `getApplicationNamespace` and made it `public` (it was package visible).